### ISSUE TITLE
Clicking BACK link does nothing

### ISSUE THREAD

**Issue by phillc73** [BlueTeck/kanboard_chrome_extension#14]
Steps to Reproduce:

1. From the main Project List, click the Clock icon to go to the Tasks page.

2. At the bottom of the Tasks page is a link labeled "BACK"

3. Click this link. No change.

This is generally true for all BACK links on all page.

Expectation was that clicking BACK would return user to Projects List page.

Browser: Chrome Version 45.0.2454.101 m

**Comment by BlueTeck** [comment]
... you are right, I will fix it.

**Comment by BlueTeck** [comment]
You could temporary fix it, if you change the switch in the settings, and change it back.

**Comment by phillc73** [comment]
I can confirm this trick does temporarily fix the issue. 

However, there was also some other strange behaviour, which I won't go into too deeply, as your fix might rectify anyway. However, when I was viewing a "Column" from my board, if I clicked the Back button at the bottom of the column, it always took me back to the same column.

e.g. I have 4 columns - A, B, C, D

I am viewing column A, click back and I'm taken to column B (which I did view earlier).
I am viewing column C, click back and I'm taken to column B
etc

I didn't take much notice of the prior steps, and I have more than 4 columns in my actual project.
